lib/main.dart

```diff
@@ -107,29 +87,9 @@ class _MainPageState extends State<MainPage> {
   }
 
   void _getChangeBottomNav(int index) {
-    /// Only one fuction to update the index
     if (index <= 4)
       context.cubit<BottomNavCubit>().updateIndex(index);
```

Comment:
this code it's same 
you can remove the condition of check index range, and just add 
`context.cubit<BottomNavCubit>().updateIndex(index);`